### PR TITLE
St/evm 787

### PR DESCRIPTION
**Pull request by saratnt**

### CODE REVIEW

File: sdk/src/main/scala/io/horizen/account/api/rpc/service/EthService.scala [hunk 1]
```diff
@@ -915,21 +915,21 @@ class EthService(
   }
 
   @RpcMethod("zen_getForwardTransfers")
-  def getForwardTransfers(hash: Hash): ForwardTransfersView = {
+  def getForwardTransfers(tag: String): ForwardTransfersView = {
     applyOnAccountView { nodeView =>
       nodeView.history
-        .getStorageBlockById(bytesToId(hash.toBytes))
+        .getStorageBlockById(getBlockIdByHashOrTag(nodeView, tag))
```
Comment (i-Alex) [inline sdk/src/main/scala/io/horizen/account/api/rpc/service/EthService.scala:921]: actually we don't want to use `tags`, only hash or height
same for zen_getFeePayments

**Review by i-Alex — CHANGES_REQUESTED** [review]

File: sdk/src/main/scala/io/horizen/account/api/rpc/service/EthService.scala (hunk 1 above)
Comment (saratnt) [reply sdk/src/main/scala/io/horizen/account/api/rpc/service/EthService.scala]: Done

File: sdk/src/main/resources/account/rpc/api/eth_openrpc.json [hunk 2]
```diff
@@ -357,7 +343,7 @@
 					"name": "Block",
 					"required": true,
 					"schema": {
-						"title": "Block number or tag",
+						"title": "Block number, block hash or tag",
```
Comment (i-Alex) [inline sdk/src/main/resources/account/rpc/api/eth_openrpc.json:346]: update the description, remove `tag`
check other places

**Review by i-Alex — CHANGES_REQUESTED** [review]

File: sdk/src/main/resources/account/rpc/api/eth_openrpc.json (hunk 2 above)
Comment (saratnt) [reply sdk/src/main/resources/account/rpc/api/eth_openrpc.json:346]: Done

**Review by i-Alex — APPROVED** [review]

**Pull request merged**